lib/resources/geo-extracts.js

```diff
@@ -0,0 +1,50 @@
+// Copyright 2025 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const { getOrNotFound } = require('../util/promise');
+const { Form } = require('../model/frames');
+const { Sanitize } = require('../util/param-sanitize');
+
+
+module.exports = (service, endpoint) => {
+
+  service.get('/projects/:projectId/forms/:xmlFormId/submissions.geojson', endpoint(async ({ Forms, GeoExtracts }, { auth, query, params }) => {
+
+    const form = await Forms.getByProjectAndXmlFormId(params.projectId, params.xmlFormId, Form.WithoutDef, Form.WithoutXml)
+      .then(getOrNotFound)
+      .then((foundForm) => auth.canOrReject('submission.list', foundForm));
+
+    return GeoExtracts.getSubmissionFeatureCollectionGeoJson(
+      form.id,
+      Sanitize.queryParamToArray(query.fieldpath),
+      Sanitize.queryParamToIntArray(query.submitterId, 'submitterId'),
+      Sanitize.getTSTZRangeFromQueryParams(query),
+      Sanitize.getSubmissionReviewStates(query.reviewState, 'reviewState'),
+      query.deleted === 'true',
```

Comment:
```suggestion
      isTrue(query.deleted),
```

For some reason, we generally support both `foo=true` and `foo=TRUE` in the API.